### PR TITLE
New inventory plugin

### PR DESCRIPTION
**Pull request by t-woerner**
The inventory plugin compiles a dynamic inventory from IPA domain, filters servers by role(s).

Usage:

Create yml file, for example `freeipa.yml`:

    ---
    plugin: freeipa
    server: server.ipa.local
    ipaadmin_password: SomeADMINpassword
    verify: ca.crt

Get compiled inventory:

    ansible-inventory -i freeipa.yml --graph

### CODE REVIEW

File: README-inventory-plugin-freeipa.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Usage
+=====
+
+Example inventory file "freeipa.yml":
+
+```yml
+---
+plugin: freeipa
+address: server.ipa.local
```
Comment (varunmylaraiah) [inline README-inventory-plugin-freeipa.md:42]: "address" variable is used instead of "server."

**Review by varunmylaraiah — CHANGES_REQUESTED** [review]
@t-woerner 
Thank you for the pull request.
In the example playbook, the "address" variable is used instead of "server." Kindly update it accordingly.

**Review by varunmylaraiah — APPROVED** [review]
changes LGTM

File: README.md [hunk 2]
```diff
@@ -469,4 +470,9 @@ Modules in plugin/modules
 * [ipauser](README-user.md)
 * [ipavault](README-vault.md)
 
+Inventory plugins in plugin/inventory
+=====================================
+
+* [freeipa](README-inventory-plugin-freeipa.md)
+
```
Comment (rjeffman) [inline README.md:477]: The last line is related to modules and not the inventory plugin.

Either this section is moved after it, or a new section "Writing new modules" is added here.

**Review by rjeffman — CHANGES_REQUESTED** [review]

**Review by rjeffman — APPROVED** [review]
LGTM.

**Pull request merged**